src/beryl/utils.cr

```diff
@@ -0,0 +1,21 @@
+require "cgi"
+
+module Beryl
+  # Helper module that provides basic parameter extraction of information
+  # from request.
+  module Utils
+    # Combines inside *params* Hash  parameters extracted from given *request*.
+    #
+    # Will combine query parameters into the resulting Hash.
+    def self.combine_params(request, params)
```

Comment:
This might not be worth it's own module and instead can be assimilated as private method of `Router`.

Considering where `HTTP::Request` is heading on Crystal master and the replacement of CGI with `HTTP::Params`, it might be good to reduce the impact of changing this API in the future.